go/src/koding/klient/machine/mount/notify/fuse/handle.go

```diff
@@ -35,7 +36,8 @@ type FileHandle struct {
 	InodeID fuseops.InodeID
 	File    *os.File
 
-	size *int64
+	size  int64
+	write int64
```

Comment:
It's leftover from the idea to have this atomic and without a mutex. Will change this to bool.